tests/unit/test_review_orchestrator.py

```
@@ -0,0 +1,356 @@
+from src.core.contract_parser import Clause
+
+
+class TestReviewOrchestrator:
+    """Test suite for ReviewOrchestrator."""
+    
+    @pytest.fixture
+    def orchestrator(self, mock_config):
+        """Create orchestrator instance with mock config."""
+        with patch('src.core.review_orchestrator.load_config', return_value=mock_config):
+            # Mock all pipeline stages
+            with patch('src.core.review_orchestrator.ContractParser') as mock_parser:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)